backend/src/discovita/service/coach/actions/handler.py

```diff
@@ -18,10 +20,12 @@ def apply_actions(state: CoachState, actions: List[Action]) -> CoachState:
     for action in actions:
         if action.type == ActionType.CREATE_IDENTITY:
             params = CreateIdentityParams(**action.params)
+            identity_id = str(uuid4())
             new_state.identities.append(Identity(
-                id=str(uuid4()),
+                id=identity_id,
                 description=params.description,
-                is_accepted=False
+                state=IdentityState.PROPOSED,
+                notes=[params.note]
```

Comment:
@caseywschmid 
Identities now have a `state` (see IdentityState enum) and `notes`.    

This does break your front-end accept-identity button, but this is working super-cleanly.